lstchain/onsite.py

```diff
@@ -59,11 +60,19 @@ def create_pro_symlink(output_dir):
     create_symlink_overwrite(pro_link, output_dir.relative_to(pro_link.parent))
 
 
+def rglob_symlinks(path, pattern):
+    """
+    Same as Path.rglob, but actually following symlinks, needed to find R0G files
+    """
+    # convert results back to path
+    return (Path(p) for p in glob(f'{path}/**/{pattern}'))
+
+
 def find_r0_subrun(run, sub_run, r0_dir=DEFAULT_R0_PATH):
     '''
     Find the given subrun R0 file (i.e. globbing for the date part)
     '''
-    file_list = r0_dir.rglob(f'LST-1.1.Run{run:05d}.{sub_run:04d}*.fits.fz')
+    file_list = rglob_symlinks(r0_dir, f'LST-1.1.Run{run:05d}.{sub_run:04d}*.fits.fz')
```

Comment:
this may change several of the calls from lstosa people, maybe they can comment on this and if the change is not absolutely necessary, we could skip it or at least make them aware of the update needed because of this.

Reply:
Its already merged into main: #1080

Reply:
ah ok, then let's leave it there then :)